app/sections/view-roadmap/macro.test.js

```diff
@@ -86,6 +86,32 @@ describe('view-roadmap', () => {
           done();
         });
     });
+
+    it('should render the document link', (done) => {
+      const context = {
+        params: {
+          section: {
+            answers: {
+              documentLink: '/solution/10001/document/roadmap.pdf',
```

Comment:
Sorry Stu, I should have mentioned that this should also change too. We use the hyphenated case as keys for the sectionIds and questionIds.